packages/keystatic/src/form/fields/number/ui.tsx

```diff
@@ -0,0 +1,39 @@
+import { NumberField } from '@keystar/ui/number-field';
+import { useReducer } from 'react';
+import { validateNumber } from './validateNumber';
+import { FormFieldInputProps } from '../../api';
+
+export function NumberFieldInput(
```

Comment:
You need to add to https://github.com/Thinkmill/keystatic/blob/ad59430dec1ed993dd3f2b6af262a9f76b1fbd02/packages/keystatic/src/form/fields/empty-field-ui.tsx